knowledge/models/res_config.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2004-2009 Tiny SPRL (<http://tiny.be>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class KnowledgeConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    module_document = fields.Boolean(
```

Comment:
In 13.0 the module is called: attachment_indexation

Reply:
Good catch!

Reply:
:+1: Done